libs/notifications/package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "@novu/notifications",
+  "version": "1.0.0",
+  "description": "Novu notification templates and workflows",
+  "main": "build/main/index.js",
+  "typings": "build/main/index.d.ts",
+  "module": "build/module/index.js",
+  "private": true,
+  "license": "MIT",
+  "keywords": [],
+  "scripts": {
+    "build": "run-p build:*",
+    "build:main": "tsc -p tsconfig.json",
+    "build:module": "tsc -p tsconfig.module.json",
+    "fix": "run-s fix:*",
+    "fix:prettier": "prettier \"src/**/*.ts\" --write",
+    "fix:lint": "eslint src --ext .ts --fix",
+    "test": "jest --passWithNoTests",
+    "lint": "eslint src",
+    "test:watch": "jest --watch",
+    "watch:build": "tsc -p tsconfig.json -w",
+    "reset-hard": "git clean -dfx && git reset --hard && pnpm install",
+    "start:studio": "npx novu@latest dev --port 3000 --route /v1/api/novu"
```

Comment:
Could we point to the local monorepo `novu` CLI package command instead?

Reply:
Sure!